content/_includes/page/base.njk

```diff
@@ -8,6 +8,7 @@
 
 {# is this page itself an entry that can be syndicated in a feed? #}
 {%- set is_entry = tags and ('_post' in tags) -%}
+{% set page_url = page.url %}
```

Comment:
I take it this is established here for use in other files? I'm not sure it gains us much.